### PR TITLE
Create `wp config get` to list configurations of `wp-config.php`

### LINKED ISSUE #2: Create `wp config get` to list configuration defined in parsed wp-config.php
As a WordPress developer, it would be helpful to have a `wp config get` command to list configuration defined in parsed wp-config.php

This command could contain both constants and PHP global variables defined in `wp-config.php`. We can take a similar approach to `WP_CLI\Runner` in identifying this data:

```
// Load wp-config.php code, in the global scope
$wp_cli_original_defined_vars = get_defined_vars();
eval( $this->get_wp_config_code() );
foreach( get_defined_vars() as $key => $var ) {
	if ( array_key_exists( $key, $wp_cli_original_defined_vars ) || 'wp_cli_original_defined_vars' === $key ) {
		continue;
	}
	global $$key;
	$$key = $var;
}
```

Previously https://github.com/wp-cli/wp-cli/issues/3831

### PR DESCRIPTION
**Pull request by rahul3883**
We can also format the result using `--format` option.
eg. `wp config get --format=csv`

Fixes #2

### CODE REVIEW

File: src/Config_Command.php [hunk 1]
```diff
@@ -166,6 +166,100 @@ public function path() {
...
+		$wp_config_vars      = self::get_wp_config_vars( get_defined_vars(), $wp_cli_original_defined_vars, array( 'wp_cli_original_defined_vars' ) );
+		$wp_config_constants = self::get_wp_config_vars( get_defined_constants(), $wp_cli_original_defined_constants );
+
+		$formatter = new \WP_CLI\Formatter( $assoc_args );
+
+		WP_CLI::line( preg_replace( '/^([A-Z ]+)/m', WP_CLI::colorize( '%9\1%n' ), "VARIABLES\n" ) );
+		$formatter->display_items( $wp_config_vars );
+
+		WP_CLI::line( preg_replace( '/^([A-Z ]+)/m', WP_CLI::colorize( '%9\1%n' ), "\nCONSTANTS\n" ) );
+		$formatter->display_items( $wp_config_constants );
```
Comment (danielbachhuber) [inline src/Config_Command.php]: Let's consolidate this to just one table, and introduce a third `type` column that denotes whether it's a `variable` or `constant`

If it's just one call to `display_items()`, the `--format=json`, etc. will produce machine-parsable output. If you call `display_items()` more than once, the rendered JSON isn't easily digestible.

Comment (rahul3883) [inline src/Config_Command.php]: > Let's consolidate this to just one table, and introduce a third type column that denotes whether it's a variable or constant

Sounds good. Done.

>If it's just one call to display_items(), the --format=json, etc. will produce machine-parsable output. If you call display_items() more than once, the rendered JSON isn't easily digestible.

👍

**Comment by rahul3883** [comment]
> you'll need to add a functional test too.

I believe a functional test should be the last part. So I was waiting for your reply about any changes if needed 😅

**Comment by danielbachhuber** [comment]
Looks great! Thanks for your work :)

**Pull request merged**